UPGRADE-4.4.md

```diff
@@ -41,6 +41,18 @@ DependencyInjection
        arguments: [!tagged_iterator app.handler]
    ```
 
+ * Passing an instance of `Symfony\Component\DependencyInjection\Parameter` as class name to `Symfony\Component\DependencyInjection\Definition` is now deprecated.
```

Comment:
"now" should be replaced by the corresponding version or completely removed.

Reply:
Adding a version number to a file called `UPGRADE-4.4.md` feels redundant. ;-)